features/step_definitions/brief_response_steps.rb

```diff
@@ -32,29 +30,28 @@
 
 Given /^I am logged in as the buyer of a closed brief with responses$/ do
   submitted_brief_response = get_brief_responses('digital-outcomes-and-specialists-2', 'submitted', 'closed').sample
-  closed_brief = get_brief(submitted_brief_response['brief']['id'])
-  @brief_id = closed_brief['id']
-  @lot_slug = closed_brief['lotSlug']
-  @framework_slug = closed_brief['frameworkSlug']
-  @buyer = closed_brief['users'][0]
+  @brief = get_brief(submitted_brief_response['brief']['id'])
+  @lot_slug = @brief['lotSlug']
+  @framework_slug = @brief['frameworkSlug']
+  @buyer = @brief['users'][0]
   @buyer.update({'password' => ENV["DM_PRODUCTION_BUYER_USER_PASSWORD"]})
   steps %Q{
     Given that buyer is logged in
   }
 end
 
 Given /^I go to that brief page$/ do
-  url = "/digital-outcomes-and-specialists/opportunities/#{@brief_id}"
+  url = "/digital-outcomes-and-specialists/opportunities/#{@brief['id']}"
```

Comment:
Ahh, nifty, I was wondering if we didn't use breif_id somewhere else, but I see you took care of it all :). Well done!